### PR TITLE
Document OES_draw_buffers_indexed

### PR DESCRIPTION
**Pull request by Elchi3**
### Description

This documents the WebGL extension https://registry.khronos.org/webgl/extensions/OES_draw_buffers_indexed/

### Motivation

https://github.com/openwebdocs/project/issues/152 is a project in which Open Web Docs is trying to provide docs for all web platform features supported in all three major engines (Webkit, Gecko, Chromium).

### Additional details

None.

### CODE REVIEW

File: files/en-us/web/api/oes_draw_buffers_indexed/blendfuncseparateioes/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+slug: Web/API/OES_draw_buffers_indexed/blendFuncSeparateiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.blendFuncSeparateiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `blendFuncSeparateiOES()` method of the `OES_draw_buffers_indexed` WebGL extension defines which function is used when blending pixels for RGB and alpha components separately for a particular draw buffer.
+
+See {{DOMxRef("OES_draw_buffers_indexed.blendFunciOES()")}} for setting RBG and alpha together and {{DOMxRef("WebGLRenderingContext.blendFuncSeparate()")}} for the WebGL 1 version of this method.
```
Comment (dawei-wang) [inline files/en-us/web/api/oes_draw_buffers_indexed/blendfuncseparateioes/index.md:12]:
```suggestion
See {{DOMxRef("OES_draw_buffers_indexed.blendFunciOES()")}} for setting RGB and alpha together and {{DOMxRef("WebGLRenderingContext.blendFuncSeparate()")}} for the WebGL 1 version of this method.
```

**Review by dawei-wang — COMMENTED** [review]
A few typos

File: files/en-us/web/api/oes_draw_buffers_indexed/blendequationseparateioes/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: OES_draw_buffers_indexed.blendEquationSeparateiOES()
+slug: Web/API/OES_draw_buffers_indexed/blendEquationSeparateiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.blendEquationSeparateiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `blendEquationSeparateiOES()` method of the `OES_draw_buffers_indexed` WebGL extension sets the RGB blend equation and alpha blend equation separately for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/blendequationseparateioes/index.md:10]:
```suggestion
The `blendEquationSeparateiOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension sets the RGB and alpha blend equations separately for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/blendfuncioes/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+---
+title: OES_draw_buffers_indexed.blendFunciOES()
+slug: Web/API/OES_draw_buffers_indexed/blendFunciOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.blendFunciOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `blendFunciOES()` method of the `OES_draw_buffers_indexed` WebGL extension defines which function is used when blending pixels for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/blendfuncioes/index.md:10]:
```suggestion
The `blendFunciOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension defines which function is used when blending pixels for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/blendfuncseparateioes/index.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+title: OES_draw_buffers_indexed.blendFuncSeparateiOES()
+slug: Web/API/OES_draw_buffers_indexed/blendFuncSeparateiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.blendFuncSeparateiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `blendFuncSeparateiOES()` method of the `OES_draw_buffers_indexed` WebGL extension defines which function is used when blending pixels for RGB and alpha components separately for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/blendfuncseparateioes/index.md:10]:
```suggestion
The `blendFuncSeparateiOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension defines which function is used when blending pixels for RGB and alpha components separately for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/colormaskioes/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+---
+title: OES_draw_buffers_indexed.colorMaskiOES()
+slug: Web/API/OES_draw_buffers_indexed/colorMaskiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.colorMaskiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `colorMaskiOES()` method of the `OES_draw_buffers_indexed` WebGL extension sets which color components to enable or to disable when drawing or rendering for a particular draw buffer. It's the indexed version of WebGL 1's {{domxref("WebGLRenderingContext.colorMask()")}} method.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/colormaskioes/index.md:10]:
```suggestion
The `colorMaskiOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension sets which color components to enable or to disable when drawing or rendering for a particular draw buffer. It's the indexed version of WebGL 1's {{domxref("WebGLRenderingContext.colorMask()")}} method.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/disableioes/index.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+---
+title: OES_draw_buffers_indexed.disableiOES()
+slug: Web/API/OES_draw_buffers_indexed/disableiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.disableiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `disableiOES()` method of the `OES_draw_buffers_indexed` WebGL extension enables blending for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/disableioes/index.md:10]:
```suggestion
The `disableiOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension enables blending for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/enableioes/index.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+---
+title: OES_draw_buffers_indexed.enableiOES()
+slug: Web/API/OES_draw_buffers_indexed/enableiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.enableiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `enableiOES()` method of the `OES_draw_buffers_indexed` WebGL extension enables blending for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/enableioes/index.md:10]:
```suggestion
The `enableiOES()` method of the {{DOMxRef("OES_draw_buffers_indexed")}} WebGL extension enables blending for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/index.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
+---
+title: OES_draw_buffers_indexed
+slug: Web/API/OES_draw_buffers_indexed
+page-type: web-api-interface
+browser-compat: api.OES_draw_buffers_indexed
+---
+{{APIRef("WebGL")}}
+
+The **`OES_draw_buffers_indexed`** extension is part of the [WebGL API](/en-US/docs/Web/API/WebGL_API) and enables use of different blend options when writing to multiple color buffers simultaneously.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/index.md:9]:
```suggestion
The **`OES_draw_buffers_indexed`** extension is part of the [WebGL API](/en-US/docs/Web/API/WebGL_API) and enables the use of different blend options when writing to multiple color buffers simultaneously.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/index.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+WebGL extensions are available using the {{domxref("WebGLRenderingContext.getExtension()")}} method. For more information, see also [Using Extensions](/en-US/docs/Web/API/WebGL_API/Using_Extensions) in the [WebGL tutorial](/en-US/docs/Web/API/WebGL_API/Tutorial).
+
+> **Note:** This extension is only available to {{domxref("WebGL2RenderingContext", "WebGL2", "", 1)}} contexts.
+
+## Instance methods
+
+- {{DOMxRef("OES_draw_buffers_indexed.blendEquationiOES()")}}
+  - : Sets both the RGB blend equation and alpha blend equation for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/index.md:18]:
```suggestion
  - : Sets both the RGB and alpha blend equations for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/index.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+- {{DOMxRef("OES_draw_buffers_indexed.blendEquationSeparateiOES()")}}
+  - : Sets the RGB blend equation and alpha blend equation separately for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/index.md:20]:
```suggestion
  - : Sets the RGB and alpha blend equations separately for a particular draw buffer.
```

File: files/en-us/web/api/oes_draw_buffers_indexed/blendequationioes/index.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
+---
+title: OES_draw_buffers_indexed.blendEquationiOES()
+slug: Web/API/OES_draw_buffers_indexed/blendEquationiOES
+page-type: web-api-instance-method
+browser-compat: api.OES_draw_buffers_indexed.blendEquationiOES 
+---
+
+{{APIRef("WebGL")}}
+
+The `blendEquationiOES()` method of the `OES_draw_buffers_indexed` WebGL extension sets both the RGB blend equation and alpha blend equation for a particular draw buffer.
```
Comment (teoli2003) [inline files/en-us/web/api/oes_draw_buffers_indexed/blendequationioes/index.md:10]:
```suggestion
The `blendEquationiOES()` method of the `OES_draw_buffers_indexed` WebGL extension sets both the RGB blend and alpha blend equations for a particular draw buffer.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few nits and this is good to go!

**Review by teoli2003 — APPROVED** [review]
👍 Thank you!

**Pull request merged**